doc/_resource_types/ec2.md

```diff
@@ -134,7 +134,7 @@ describe ec2('my-ec2') do
 end
 ```
 
-### advanced
+### Advanced
```

Comment:
Could you use `advanced` for auto document generation ?

Reply:
Done!